test/views/DateGoodsUnloadedRejectionViewSpec.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.{AppWithDefaultMockFixtures, SpecBase}
+import forms.DateGoodsUnloadedFormProvider
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import views.behaviours.DateInputViewBehaviour
+import views.html.DateGoodsUnloadedRejectionView
+
+import java.time.{Clock, LocalDate}
+
+class DateGoodsUnloadedRejectionViewSpec extends SpecBase with DateInputViewBehaviour with AppWithDefaultMockFixtures {
```

Comment:
Should just need to extend `DateInputViewBehaviour` here I believe

Reply:
Done